posthog/temporal/llm_analytics/trace_clustering/coordinator.py

```diff
@@ -0,0 +1,222 @@
+"""
+Coordinator workflow for daily trace clustering.
+
+This workflow discovers teams with trace embeddings and spawns
+child workflows to cluster traces for each team.
+"""
+
+import dataclasses
+from datetime import timedelta
+from typing import Any
+
+import structlog
+import temporalio
+
+from posthog.sync import database_sync_to_async
+from posthog.temporal.common.base import PostHogWorkflow
+from posthog.temporal.llm_analytics.trace_clustering import constants
+from posthog.temporal.llm_analytics.trace_clustering.models import ClusteringWorkflowInputs
+from posthog.temporal.llm_analytics.trace_clustering.workflow import DailyTraceClusteringWorkflow
+
+logger = structlog.get_logger(__name__)
+
+
+@dataclasses.dataclass
+class TraceClusteringCoordinatorInputs:
+    """Inputs for the coordinator workflow."""
+
+    lookback_days: int = constants.DEFAULT_LOOKBACK_DAYS
+    max_samples: int = constants.DEFAULT_MAX_SAMPLES
+    min_k: int = constants.DEFAULT_MIN_K
+    max_k: int = constants.DEFAULT_MAX_K
+    min_embeddings: int = constants.MIN_TRACES_FOR_CLUSTERING
+    max_concurrent_teams: int = constants.DEFAULT_MAX_CONCURRENT_TEAMS
+
+
+@dataclasses.dataclass
+class TeamsWithEmbeddingsResult:
+    """Result from querying teams with embeddings."""
+
+    team_ids: list[int]
+
+
+def query_teams_with_embeddings(lookback_days: int, min_embeddings: int) -> list[int]:
+    """
+    Query ClickHouse for teams that have trace embeddings in the lookback window.
+
+    Shared logic used by both the coordinator activity and manual trigger script.
+    """
+    from django.utils import timezone
+
+    from posthog.clickhouse.client.connection import Workload
+    from posthog.clickhouse.client.execute import sync_execute
+
+    end_dt = timezone.now()
+    start_dt = end_dt - timedelta(days=lookback_days)
+
+    result = sync_execute(
+        """
+        SELECT
+            team_id,
+            count(DISTINCT document_id) as embedding_count
+        FROM posthog_document_embeddings
```

Comment:
@oliverb123 👋 - just wanted to double check am i ok to be querying this `posthog_document_embeddings` of should i be using `distributed_posthog_document_embeddings` or something - just double checking if `posthog_document_embeddings` is still correct table to be reading from in production

Reply:
hmm i see - i can refactor to do this.